### PR TITLE
Update FAQ page

### PR DESCRIPTION
**Pull request by csaye**
- Makes all faq arrows same color
- Updates faq content

### CODE REVIEW

File: src/components/FAQ/Question.jsx [hunk 1]
```diff
@@ -5,7 +5,6 @@ import "../../index.css";
 import { CopyP } from "../../utility/ContentStyles.js";
```
Comment (KindaOK) [inline src/components/FAQ/Question.jsx]: Can you fix any relative imports in the file while you're at it?

File: src/components/FAQ/FAQ.jsx [hunk 2]
```diff
@@ -12,59 +12,50 @@ const FAQ = () => (
             <br/>
```
Comment (KindaOK) [inline src/components/FAQ/FAQ.jsx:12]: Same case in this file: fix any relative imports.

**Review by KindaOK — CHANGES_REQUESTED** [review]
Replace all the (unreasonably deep) relative imports with absolute imports

**Comment by csaye** [comment]
Fixed all relative imports.

**Review by KindaOK — APPROVED** [review]
lgtm

**Pull request merged**